<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources.qrc">
    <normaloff>:/images/liveTable</normaloff>:/images/liveTable</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <property name="icon">
      <iconset resource="../../Engine/EResources.qrc">
       <normaloff>:/images/language</normaloff>:/images/language</iconset>
     </property>
     <addaction name="actionEnglishTr"/>
     <addaction name="actionPortugueseTr"/>
     <addaction name="actionChineseTr"/>
    </widget>
    <addaction name="hostRoomAction"/>
    <addaction name="joinRoomAction"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Table"/>
    <addaction name="leaveRoomAction"/>
    <addaction name="separator"/>
    <addaction name="menuLanguage"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuRoom">
    <property name="title">
     <string>Room</string>
    </property>
    <addaction name="throwDiceAction"/>
    <addaction name="addObjectAction"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionRotate_Left"/>
    <addaction name="actionRotate_Right"/>
    <addaction name="separator"/>
    <addaction name="actionRoom_Info"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRoom"/>
  </widget>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/quit</normaloff>:/images/quit</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="joinRoomAction">
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/find</normaloff>:/images/find</iconset>
   </property>
   <property name="text">
    <string>Join Room</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="hostRoomAction">
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/create</normaloff>:/images/create</iconset>
   </property>
   <property name="text">
    <string>Host Room</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="addObjectAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/add</normaloff>:/images/add</iconset>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="throwDiceAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/dice</normaloff>:/images/dice</iconset>
   </property>
   <property name="text">
    <string>Throw Dice</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="leaveRoomAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/leave</normaloff>:/images/leave</iconset>
   </property>
   <property name="text">
    <string>Leave Room</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/settings</normaloff>:/images/settings</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionEnglishTr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/flags/english</normaloff>:/images/flags/english</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionPortugueseTr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/flags/portuguese</normaloff>:/images/flags/portuguese</iconset>
   </property>
   <property name="text">
    <string>Portuguese</string>
   </property>
  </action>
  <action name="actionChineseTr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/flags/chinese</normaloff>:/images/flags/chinese</iconset>
   </property>
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/sZoomIn</normaloff>:/images/sZoomIn</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/sZoomOut</normaloff>:/images/sZoomOut</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionRotate_Left">
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/rotateLeft</normaloff>:/images/rotateLeft</iconset>
   </property>
   <property name="text">
    <string>Rotate Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionRotate_Right">
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/rotateRight</normaloff>:/images/rotateRight</iconset>
   </property>
   <property name="text">
    <string>Rotate Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionSave_Table">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/save</normaloff>:/images/save</iconset>
   </property>
   <property name="text">
    <string>Save Table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRoom_Info">
   <property name="icon">
    <iconset resource="../../Engine/EResources.qrc">
     <normaloff>:/images/info</normaloff>:/images/info</iconset>
   </property>
   <property name="text">
    <string>Room Info</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources.qrc"/>
  <include location="../../Engine/EResources.qrc"/>
 </resources>
 <connections/>
</ui>
